README.rst

```diff
@@ -58,12 +58,164 @@ Python:
   provides an abstraction of NI-DAQmx in the ``nidaqmx`` module, which collides
   with this package's module name.
 
+Getting Started
+===============
+
+In order to use the **nidaqmx** package, you must have at least one DAQ device installed on your
+machine. X series DAQ device (eg: PCIe-6363 or USB-6351) is recommended, physical or
+simulated devices are supported. You can use the **NI MAX** or **NI Hardware Configuration Utility**
+to verify and configure your devices.
+
+Verify and configure devices on **NI MAX**:
+
+.. image:: max_device_name.png
+  :alt: NI MAX Device Name
+  :align: center
+  :width: 800px
+
+Verify and configure devices on **NI Hardware Configuration Utility**:
+
+.. image:: hwcu_device_name.png
+  :alt: NI HWCU Device Name
+  :align: center
+  :width: 800px
+
+Virtual channels and tasks are fundamental components of NI-DAQmx.
+
+Task in NI-DAQmx
+----------------
+A task is a collection of one or more virtual channels with timing, triggering, and other properties.
+Refer to the `NI-DAQmx Task <https://www.ni.com/docs/en-US/bundle/ni-daqmx/page/tasksnidaqmx.html>`_ for more information.
+
+Example code to create a task:
+
+.. code-block:: python
+
+  >>> import nidaqmx
+  >>> with nidaqmx.Task() as task:
+  ...     pass
+
+Virtual Channels in NI-DAQmx
+----------------------------
+Virtual channels, or sometimes referred to generically as channels, are software entities that encapsulate the physical channel
```

Comment:
Ok, what's a physical channel then? The content on the linked page introduces them, but you didn't copy that.